roles/common/tasks/container_tests.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: Get container status
+  ansible.builtin.shell:
+    cmd: |
+      podman ps -a --format "{{ '{{.Names}} {{.Status}}' }}" | grep "{{ item }}" | awk '{print $2;}'
```

Comment:
For readability, I recommend using something instead of item.

You can use a useful variable name, and tell the loop in main to use that same var name for its loop var.

The syntax would be similar to: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#stacking-loops-via-include-tasks, but without the nested loops. 
I would suggest container_name as the loop var here, so that reading this task is easier.

Here's an example playbook:
```
---
- hosts: localhost
  tasks:
    - ansible.builtin.debug:
        msg: "{{ item }}"
      loop:
        - "first"
        - "second"

    - ansible.builtin.debug:
        msg: "{{ my_var }}"
      loop:
        - "third"
        - "fourth"
      loop_control:
        loop_var: my_var
```